src/components/cart/CartHeader.ts

```diff
@@ -36,7 +36,7 @@ class CartHeader extends Base<HTMLDivElement> {
 
   private updateCartTitle(): void {
     const count = this.store.countCartItems();
-    this.title.textContent = `Cart (${count})`;
+    this.title.textContent = `${this.store.getOptions().cart?.titleLabel ?? 'Cart'} (${count})`;
```

Comment:
I think `title` here would be enough as property name. Also I would move some logic out:

```
const {cart} = this.store.getOptions();
this.title.textContent = `${cart?.title ?? 'Cart'} (${count})`;
```